pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DdpKit.java

```diff
@@ -44,14 +44,13 @@ public DdpKit(String dsmKitRequestId, String kitLabel, String trackingToId, Stri
         this.upsTrackingDate = upsTrackingDate;
         this.upsReturnStatus = upsReturnStatus;
         this.upsReturnDate = upsReturnDate;
-        this.HRUID = bspCollaboratodId;
+        this.hruid = bspCollaboratorId;
```

Comment:
@pegahtah does that make sense? bspCollaboratorId != hruid for most customers

Reply:
The class was only for Tbos  cloud function